src/components/FullscreenLoadingIndicator.js

```diff
@@ -5,17 +5,22 @@ import {ActivityIndicator, StyleSheet, View} from 'react-native';
 import styles from '../styles/styles';
 import themeColors from '../styles/themes/default';
 import stylePropTypes from '../styles/stylePropTypes';
+import Log from '../libs/Log';
 
 const propTypes = {
-    /** Controls whether the loader is mounted and displayed */
-    visible: PropTypes.bool,
+    /** Name used in the logs if the loader is displayed for too long time */
+    name: PropTypes.string,
```

Comment:
**suggestion:** `logDetail` or `logContent` to make it clear what the `name` will be used for